src/main/java/com/revature/validation/ControllerValidationHandler.java

```diff
@@ -0,0 +1,31 @@
+package com.revature.validation;
+
+import javassist.NotFoundException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'javassist.NotFoundException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)